programmers/난이도별/level02.가장_큰_수/s2uyeoii.cpp

```diff
@@ -0,0 +1,52 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+
+//주어진 원소 숫자 비교
+int str_compare(int* num1, int* num2) {
+    char num_cmp1[10] = ""; //0~10까지 
+    char num_cmp2[10] = "";
+
+    sprintf(num_cmp1, "%d%d", *num1, *num2); //주어진 문자 문자열 저장
+    sprintf(num_cmp2, "%d%d", *num2, *num1);
+
+    if (strcmp(num_cmp1, num_cmp2) < 0) return 1; 
+    else if (strcmp(num_cmp1, num_cmp2) == 0) return 0;
+    else return -1;
```

Comment:
```suggestion
    if (strcmp(num_cmp1, num_cmp2) < 0) return 1; 
    if (strcmp(num_cmp1, num_cmp2) == 0) return 0;
    return -1;
```
바로 return 하기 때문에 else는 없어도 무방할 것 같아요!

Reply:
``` c
return strcmp(num_cmp2, num_cmp1);
```
`strcmp`의 반환값을 생각하면 3줄의 if문을 하나의 리턴문으로 합쳐 작성할수도 있습니다.